ml_garden/core/model.py

```diff
@@ -3,11 +3,13 @@
 
 import pandas as pd
 
+from ml_garden.core.constants import Task
+
 
 class Model(ABC):
     """Base class for models."""
 
-    TASK = ""
+    TASKS: List[Task] = []
```

Comment:
This was a simple `str` before. Are you sure you need a `List[Task]` or just `Task`?

Reply:
For most cases I need just a Task. For example: XGBRegressor or XGBClassifier.

But I was adding support for [AutoGluon](https://auto.gluon.ai/stable/index.html) on another PR and that is an auto model that works with more than one Task. I don't know if there are more models that support more than one task but it was my way for fixing this for AutoGluon